### PR TITLE
Backports 0.18 pr10

### PR DESCRIPTION
**Pull request by Munkybooty**

### CODE REVIEW

File: doc/README.md [hunk 1]
```diff
@@ -2,6 +2,11 @@ Dash Core
 ==========
 
 This is the official reference wallet for Dash digital currency and comprises the backbone of the Dash peer-to-peer network. You can [download Dash Core](https://www.dash.org/downloads/) or [build it yourself](#building) using the guides below.
+Setup
+---------------------
+Dash Core is the original Dash client and it builds the backbone of the network. It downloads and, by default, stores the entire history of Dash transactions, which requires a few hundred gigabytes of disk space. Depending on the speed of your computer and network connection, the synchronization process can take anywhere from a few hours to a day or more.
+
+To download Dash Core, visit [dash.org](https://www.dash.org/downloads/).
```
Comment (UdjinM6) [inline doc/README.md:9]: let's mark 14511 as DNM

**Review by UdjinM6 — CHANGES_REQUESTED** [review]
14424 breaks psbt tests (should find follow up fixes or drop it for now) + see below + 96aaa411105a02baf41268d1906d240c8497a57f + 19503424a3bd0dd63344355236fa8b82cd2ce480

we have pr9 already (#4278), renamed to pr10

File: src/rpc/rawtransaction.cpp [hunk 2]
```diff
@@ -509,12 +509,14 @@ static UniValue createrawtransaction(const JSONRPCRequest& request)
...
             "    {\n"
-            "      \"data\": \"hex\"        (obj, optional) A key-value pair. The key must be \"data\", the value is hex encoded data\n"
+            "      \"data\": \"hex\"        (obj, optional) A key-value pair. The key must be \"data\", the value is hex-encoded data\n"
             "    }\n"
             "    ,...                     More key-value pairs of the above form. For compatibility reasons, a dictionary, which holds the key-value pairs directly, is also\n"
             "                             accepted as second parameter.\n"
             "   ]\n"
             "3. locktime                  (numeric, optional, default=0) Raw locktime. Non-0 value also locktime-activates inputs\n"
+            "4. replaceable               (boolean, optional, default=false) Marks this transaction as BIP125-replaceable.\n"
+            "                             Allows this transaction to be replaced by a transaction with higher fees. If provided, it is an error if explicit sequence numbers are incompatible.\n"
```
[the remark this answers is not in the document]
Comment (PastaPastaPasta) [reply src/rpc/rawtransaction.cpp:519]: I don't believe this has been resolved

**Review by PastaPastaPasta — CHANGES_REQUESTED** [review]

File: src/rpc/rawtransaction.cpp (hunk 2 above)
[the remark this answers is not in the document]
Comment (PastaPastaPasta) [reply src/rpc/rawtransaction.cpp:519]: still not done :)

[the remark this answers is not in the document]
Comment (Munkybooty) [reply src/rpc/rawtransaction.cpp:519]: Could've sworn I did it

**Review by UdjinM6 — APPROVED** [review]
utACK

**Review by PastaPastaPasta — APPROVED** [review]
utACK for merging via merge commit

**Pull request merged**